### PR TITLE
[CP-881] Go to support modal isn't possible during the restore/update fails occurred by disconnection

### PR DESCRIPTION
**Pull request by dkarski**
Jira: [CP-881]

**Description**

Fixed the bug by moved contact support modal logic displaying to `modals-manager`

Small refactore
- `contact-support` domain move out to root catalog
- `help` domain move out to root catalog
- `recovery-mode` domain move out to root catalog 

[CP-881]: https://appnroll.atlassian.net/browse/CP-881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/contact-support/containers/contact-support-flow.container.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  SendTicketPayload,
+} from "App/contact-support/actions/send-ticket.action"
+import { closeContactSupportFlow } from "App/contact-support/actions/close-contact-support-flow.action"
+
+const mapStateToProps = (state: RootState & ReduxRootState) => ({
+  files: files,
+  state: state.contactSupport.state,
+})
+
+const mapDispatchToProps = (dispatch: TmpDispatch) => ({
```
Comment (igorbezsmertnyi) [inline packages/app/src/contact-support/containers/contact-support-flow.container.tsx:21]: You can use simpler construction here
```ts
const mapDispatchToProps = {
    closeContactSupportFlow, 
    sendTicket,
}
```

File: packages/app/src/contact-support/reducers/contact-support.reducer.test.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
```
Comment (igorbezsmertnyi) [inline packages/app/src/contact-support/reducers/contact-support.reducer.test.ts:1]: Let's, also write tests for other cases

**Review by wercix — APPROVED** [review]

File: packages/app/src/contact-support/reducers/contact-support.reducer.test.ts (hunk 2 above)
Comment (dkarski) [reply packages/app/src/contact-support/reducers/contact-support.reducer.test.ts:1]: https://github.com/mudita/mudita-center/pull/561/commits/1372585d5696ef8c14a7e57d166d94b1a11eb76b

File: packages/app/src/contact-support/containers/contact-support-flow.container.tsx (hunk 1 above)
Comment (dkarski) [reply packages/app/src/contact-support/containers/contact-support-flow.container.tsx]: https://github.com/mudita/mudita-center/pull/561/commits/2eee9b5cab017bb9d533a4b3b3e2138cf475ded6

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by wercix — APPROVED** [review]

**Pull request merged**